functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configuration/SkipTestExecutionFunctionalTest.java

```diff
@@ -72,4 +78,97 @@ public void should_disable_smart_testing_and_execute_no_tests_when_test_executio
         assertThat(capturedMavenLog).contains(SMART_TESTING_EXTENSION_DISABLED);
         assertThat(actualTestResults.accumulatedPerTestClass()).size().isEqualTo(0);
     }
+
+    @Test
+    public void should_disable_smart_testing_and_execute_no_tests_when_skip_test_is_set_as_default_in_pom() throws Exception {
+        // given
+        final Project project = testBed.getProject();
+
+        project
+            .applyAsCommits("Configures skipTests as default property in pom");
```

Comment:
Taking from the commit: 
```xml
+    <skipTests>true</skipTests>
   </properties>
 
   <issueManagement>
@@ -80,6 +83,7 @@
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
+          <skipTests>${skipTests}</skipTests>
```
if you use it in this way, it will be still the property defined in the plugin. If you want to set it only using properties, then the line `<skipTests>${skipTests}</skipTests>` cannot be present there.
In other words - what you wrote there has the very same effect as this one: https://github.com/hemanik/smart-testing-dogfood-repo/commit/3a5380aa90a39abb6e0f58f0539353f808cf3411

Reply:
Updated the tag.